fractal_server/app/api/v1/workflow.py

```diff
@@ -60,3 +61,20 @@ async def create_workflow(
     await db.commit()
     await db.refresh(db_workflow)
     return db_workflow
+
+
+@router.delete("/{_id}", status_code=status.HTTP_204_NO_CONTENT)
+async def delete_workflow(
+    _id: int,
+    user: User = Depends(current_active_user),
+    db: AsyncSession = Depends(get_db),
+):
```

Comment:
missing access control.
please check that user has rights on the project the workflow belongs to. raise the appropriate error otherwise